bin/install_nodejs

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
you can't replace this file with multi buildpack ? It provides node and npm, even if there is no package.json